README.md

```diff
@@ -3,3 +3,5 @@ Practicing git.
 
 Lesson 1: doing PR
 
+Lesson 2: explaining review to ....
```

Comment:
The title doesn't really explain what we are learning about. Suggestion:

**Lesson 2: step by step how to handle reviews in GitHub and provide feedback**